src/js-feature-toggle-manager.ts

```diff
@@ -7,7 +7,7 @@ export interface ToggleStatus {
 
 export class FeatureToggleManager {  
   private static _instance: FeatureToggleManager;
-  private currentToggles: ToggleStatus[];
+  private _toggles: ToggleStatus[];
```

Comment:
@Izzmo 
The problem I was trying to solve was that I want a private member variable called `toggles` and a public getter called `toggles`. If I named them the same, I got a compiler error. The TypeScript documentation showed using an underscore as a prefix, so I went with that.

But if we're moving to follow C# standards, maybe I should make the public getter capitalized to be `Toggles`, which would allow the private member to be `toggles`.

Thoughts?